scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -310,9 +314,9 @@ def generate_json(self):
             sys.exit(-1)
 
         try:
-            json_file = open(self._args.output, "w+")
+            json_file = open(self._args.output+".json", "w+")
```

Comment:
For clarity we may keep the `self._args.output+".json` as a constant named e.g. `output`/`final_output`.